quesma/model/bucket_aggregations/histogram.go

```diff
@@ -33,8 +33,8 @@ func (query Histogram) TranslateSqlResponseToJson(rows []model.QueryResultRow, l
 	var response []model.JsonMap
 	for _, row := range rows {
 		response = append(response, model.JsonMap{
-			"key":       row.Cols[level-1].Value,
-			"doc_count": row.Cols[level].Value,
+			"key":       row.Cols[len(row.Cols)-2].Value,
+			"doc_count": row.Cols[len(row.Cols)-1].Value,
```

Comment:
let's maybe switch to this way, it's already like that in e.g. `histogram` and most other aggregations.
And start worrying only if we see it fail, which doesn't seem likely with amount of tests we have

Reply:
Ideally would get right of level in many places, but probably after pancakes.